login.php

```diff
@@ -90,7 +91,7 @@
 					<h3><label for="Password">Password</label></h3>
 				</div>
 				<div class="column2">
-					<input type="text" id="Password" name="Password" placeholder="Password" style="width:350px; height:25px;">
+					<input type="password" id="Password" name="Password" placeholder="Password" style="width:350px; height:25px;">

```

Comment:
input type changed to password so that it is more functional as a password input field